### ISSUE TITLE
The change in 2.5.0 to import Babel helpers is causing all of them to be included

### ISSUE THREAD

**Issue by ericf** [rollup/rollup-plugin-babel#65]
I've noticed that all of the Babel helpers are being included in Rollup bundles since 2.5.0. This is because of the `import * as babelHelpers from '\0babelHelpers';`. Even after using the Uglify Rollup plugin, helpers that aren't being used are still there.

It seems that the `usedHelpers` metadata on the AST that Babel provides should be leveraged so that only the helpers that are actually needed are imported.

**Comment by eventualbuddha** [comment]
I believe the expectation was that rollup would treeshake out the helpers that you're not using. Do you have a reproducible example that illustrates this?

**Comment by Victorystick** [comment]
@ericf It should be just as @eventualbuddha says.

**Comment by ericf** [comment]
@eventualbuddha Okay I was able to narrow it down. When I include the `transform-es3-member-expression-literals` Babel plugin, then the tree-shaking of the Babel helpers doesn't happen.

```json
{
    "presets": [
        "es2015-rollup"
    ],
    "plugins": [
        "transform-es3-member-expression-literals"
    ]
}
```

I tried different `.babelrc` configurations with multiple presets and using `"passPerPreset": true`, but nothing seems to work to properly tree-shake the helpers when using `transform-es3-member-expression-literals`.
